### PR TITLE
hypertable_(detailed_)size functions now supports CAGG name

### PR DESCRIPTION
**Pull request by noctarius**
# Description

The two functions `hypertable_size` and `hypertable_detailed_size` now support passing the name of a continuous aggregate and transparently looking up the backing hypertable to return its statistics.

# Links

https://github.com/timescale/timescaledb/pull/5159

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: api/hypertable_detailed_size.md [hunk 1]
```diff
@@ -11,18 +11,24 @@ api:
...
+Get detailed information about disk space used by a hypertable or
+continuous aggregate, returning size information for the table
+itself, any indexes on the table, any toast tables, and the total
+size of all. All sizes are reported in bytes. If the function is
+executed on a distributed hypertable, it returns size information
+as a separate row per node, including the access node.
+
+<Highlight type="tip">
+When a continuous aggregate name is provided, the function will
+transparently look up the backing hypertable and return its statistics
```
Comment (Loquacity) [inline api/hypertable_detailed_size.md:23]:
```suggestion
When a continuous aggregate name is provided, the function
transparently looks up the backing hypertable and returns its statistics
```

File: api/hypertable_size.md [hunk 2]
```diff
@@ -11,17 +11,24 @@ api:
 
 # hypertable_size()  
 
-Get the total disk space used by a hypertable, that is, the sum of the
-size for the table itself (including chunks), any indexes on the
-table, and any toast tables. The size is reported in bytes. This is
-equivalent to computing the sum of `total_bytes` column from the
-output of `hypertable_detailed_size` function.
+Get the total disk space used by a hypertable or continuous aggregate,
+that is, the sum of the size for the table itself (including chunks),
```
Comment (Loquacity) [inline api/hypertable_size.md:15]:
```suggestion
that is, the sum of the size for the table itself including chunks,
```

File: api/hypertable_size.md [hunk 3]
```diff
@@ -11,17 +11,24 @@ api:
...
+any indexes on the table, and any toast tables. The size is reported
+in bytes. This is equivalent to computing the sum of `total_bytes`
+column from the output of `hypertable_detailed_size` function.
+
+<Highlight type="tip">
+When a continuous aggregate name is provided, the function will
+transparently look up the backing hypertable and return its statistics
```
Comment (Loquacity) [inline api/hypertable_size.md:22]:
```suggestion
When a continuous aggregate name is provided, the function
transparently looks up the backing hypertable and returns its statistics
```

File: api/hypertable_size.md [hunk 4]
```diff
@@ -51,3 +58,13 @@ Get size information for all hypertables.
 SELECT hypertable_name, hypertable_size(format('%I.%I', hypertable_schema, hypertable_name)::regclass)
   FROM timescaledb_information.hypertables;
 ```
+
+Get size of a continuous aggregate.
```
Comment (Loquacity) [inline api/hypertable_size.md:62]:
```suggestion
Get the size of a continuous aggregate:
```

Comment (noctarius) [reply api/hypertable_size.md:62]: I think in this case the other sentences should be fixed, too. I specifically copied the weird style of the two examples before 😅

```
Get size information for a hypertable.
Get size information for all hypertables.
Get size of a continuous aggregate.
```

File: api/hypertable_size.md [hunk 5]
```diff
@@ -38,7 +45,7 @@ output of `hypertable_detailed_size` function.
 Get size information for a hypertable.
```
Comment (Loquacity) [inline api/hypertable_size.md:45]:
```suggestion
Get the size information for a hypertable.
```

File: api/hypertable_size.md (hunk 4 above)
Comment (Loquacity) [reply api/hypertable_size.md:62]: the 'information' and 'for' in those earlier sentences mean that the definite article is not required. You can either make them all "Get the size of ..." or "Get size information for ...".

**Review by Loquacity — APPROVED** [review]

**Pull request merged**